docs/user/range_time.md

```diff
@@ -0,0 +1,100 @@
+# Range-Time Parameter Plots 
+---
+
+Range-time parameter plots (also known as range-time intensity (RTI) plots) plot a radar measured parameter along range gates versus time. They are the most common way to look at data from a single radar. 
+
+!!! Note
+    Slant ranges are future enhancement we are currently working on. Sorry for inconvenience.
+
+### Basic RTP
+The general syntax for plot_range_time is:
+'plot_range_time(fitacf_data, options)'
+where 'fitacf_data' is the read in data, and the options are several python parameters used to control how the plot looks.
+
+First, make sure Pydarn and matplotlib are imported, then read in the .fitacf file with the data you wish to plot:
+```python
+import pydarn
+import matplotlib.pyplot as plt
+
+fitacf_file = "path/to/fitacf/file"
+sdarn_read = pydarn.SDarnRead(fitacf_file)
+fitacf_data = sdarn_read.read_fitacf()
+
+```
+
+You can choose one of four data products to plot:
+
+| Data product                | String name |
+|-----------------------------|-------------|
+| Line of sight velocity (m/s)| 'v'         |
+| Spectral width (m/s)        | 'w_l'       |
+| Elevation angle (degrees)   | 'elv'       |
+| Power (dB)                  | 'p_l'       |
+
+which is chosen by adding: 
+'parameter=String name' 
+as an option. The default if left blank is 'v'.
+
+To specify which beam to look at, add the option:
+'beam=beam_number"
+
+As an example, taking a look at some 'v' data from the Clyde River radar, beam 7:
+```python
+fitacf_file = "20190831.C0.cly.fitacf"
+sdarn_read = pydarn.SDarnRead(fitacf_file)
+fitacf_data = sdarn_read.read_fitacf()
+
+plt.title("Clyde River Beam 7")
+pydarn.RTP.plot_range_time(fitacf_data, beam_num=7)              
```

Comment:
I'd recommend making this more user friendly and saying something like: get the beam number of the first data element by using `fitacf_data[0]['bmnum']` and the radar ID like `fitacf_data[0]['stid']`.  You can then create a title and plot the data: 
```
pydarn.RTP.plot_range_time(fitacf_data, beam_num=fitacf_data[0]['bmnum'])
plt.title("Radar {:d}, Beam {:d}".format(fitacf_data[0]['stid'], fitacf_data[0]['bmnum'])
```

Then, if they have the same file they get the example, but if they don't, they get something sensible and have a better understanding of how to perform a quicklook summary of a new data file.